src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+@import "../utils/_variables";
+
+@import "../utils/mixins";
+
+@import "../utils/normalize";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-box-size;
+  gap: $box-gap;
+  margin: 100px;
+  align-items: center;
+
+  @for $i from 1 through 31 {
+    & :nth-child(#{$i})::before {
+      content: "#{$i}";
+    }
+  }
+
+  @each $day, $dayNum in $days-of-week {
+    &--start-day--#{$day} :first-child {
+      margin-left: $dayNum * ($calendar-dey-box + $box-gap);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
```

Comment:
Would be better if you create additional file for loops

Reply:
No keep these to selectors as close as possible.